packages/react/src/vm-extension/__tests__/node-vm-ext-list.spec.tsx

```diff
@@ -0,0 +1,37 @@
+import { render, waitFor } from "@testing-library/react";
+import React from "react";
+import { translate } from "@azure/bonito-core";
+import { initMockBatchBrowserEnvironment } from "../../environment";
+import { NodeVMExtList } from "../node-vm-ext-list";
+
+describe("PoolVMExtList", () => {
+    beforeEach(() => {
+        initMockBatchBrowserEnvironment();
+    });
+
+    it("should render vm extension lists", async () => {
+        const { getByText } = render(
+            <NodeVMExtList
+                accountEndpoint="dummy_endpoint"
+                poolId={"dummy_pool"}
+                nodeId="tvmps_id1"
+            />
+        );
+        getByText(translate("lib.react.vmExtension.name"));
```

Comment:
Can we add an accessibility test in here somewhere? Also in the pool extension list test.

Reply:
added for node and pool VM extension list